registry/handlers/images.go

```diff
@@ -83,6 +91,64 @@ func (imh *imageManifestHandler) GetImageManifest(w http.ResponseWriter, r *http
 		return
 	}
 
+	supportsSchema2 := false
+	supportsManifestList := false
+	if acceptHeaders, ok := r.Header["Accept"]; ok {
```

Comment:
Make to use [`http.CanonicalHeader`](https://golang.org/pkg/net/http/#CanonicalHeaderKey) when accessing the header map directly.